engines/aio/Dockerfile

```diff
@@ -37,11 +38,11 @@ COPY target/${env.project_artifactId}-${env.project_version}.jar /usr/bin
 RUN ln /usr/bin/${env.project_artifactId}-${env.project_version}.jar /usr/bin/${env.project_artifactId}.jar && \
     yum install -y $IMAGEMAGICK_DEP_RPM_URL && \
     if [ "$(uname -m)" = "x86_64" ]; then \
-      yum install -y ${IMAGEMAGICK_RPM_URL}-el8.rpm ${IMAGEMAGICK_RPM_URL}-el8-libs.rpm; \
+      yum install -y ${IMAGEMAGICK_RPM_URL}-el9.rpm ${IMAGEMAGICK_RPM_URL}-libs-el9.rpm; \
     else \
-      yum install -y ${IMAGEMAGICK_RPM_URL}-arm64.rpm ${IMAGEMAGICK_RPM_URL}-arm64-libs.rpm; \
+      yum install -y ${IMAGEMAGICK_RPM_URL}-el9-aarch64.rpm ${IMAGEMAGICK_RPM_URL}-libs-el9-aarch64.rpm; \
     fi && \
-    yum install -y cairo cups-libs libSM libGLU && \
+    yum install -y cairo cups-libs libSM libGLU procps && \
```

Comment:
why do we need to install `procps`? 🤔

Reply:
The `ps` command is not installed by default in Rocky Linux 9 and that command is required to start the application (`org.artofsolving.jodconverter.process.LinuxProcessManager`)